static/build/.tmp_update/script/scraper/scrap_screenscraper.sh

```diff
@@ -424,18 +435,23 @@ for file in $(eval "find /mnt/SDCARD/Roms/$CurrentSystem -maxdepth 2 -type f \
         
         echo Searching on screenscraper ...
         api_result=$(curl -k -s "$url") 
-
-    	
-    
-    	
+		
         # Don't check art if screenscraper is closed
         if echo $api_result | grep -q "API closed"; then
         	echo -e "The Screenscraper API is currently down, please try again later."
         	echo -e "Press the ${RED}A button${NONE} to finish"
         	let Scrap_Fail++;
         	break;
         fi
-        
+
+        # Don't check art if max threads for leechers is used
+        if echo $api_result | grep -q "The maximum threads"; then
```

Comment:
It happens too often when the user is not logged in. I'm going to implement a retry here (something like 3 retries with a pause of 3 seconds between each before exiting).